### PR TITLE
🌐 Add Spanish translation for `docs/es/docs/advanced/security/index.md`

### PR DESCRIPTION
**Pull request by Xaraxx**
This is the translated version of the advanced/security/index.md file

### CODE REVIEW

File: docs/es/docs/advanced/security/index.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Seguridad Avanzada - Introducción
+
+## Características Adicionales
+
+Hay algunas características adicionales para manejar la seguridad además de las que se tratan en el [Tutorial - Guía de Usuario: Seguridad](../../tutorial/security/){.internal-link target=_blank}.
+
+!!! tip
+    Las secciones siguientes se consideran **no necesariamente "avanzadas"**. 
+
+    Y es posible que para tu caso de uso, la solución esté en alguno de ellos.
```
Comment (dmas-at-wiris) [inline docs/es/docs/advanced/security/index.md:10]: ... en alguna de ellas.

File: docs/es/docs/advanced/security/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+
+## Leer primero el Tutorial
+
+En las siguientes secciones asumimos que ya haz leído el principal [Tutorial - Guía de Usuario: Seguridad](../../tutorial/security/){.internal-link target=_blank}.
```
Comment (dmas-at-wiris) [inline docs/es/docs/advanced/security/index.md:15]: que ya **has** leído

**Review by pedabraham — APPROVED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit b4d2f9fe7a14845a3467b8e770c1042d897693cb at: https://bdd2be45.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
Great, thanks @Xaraxx! 🍰 

And thanks for the help @dmas-at-wiris and @pedabraham. ☕

**Pull request merged**